pkgs/stdenv/generic/check-meta.nix

```diff
@@ -113,6 +108,40 @@ let
   showLicense = showLicenseOrSourceType;
   showSourceType = showLicenseOrSourceType;
 
+  # Extract a list of warnings from the derivation. Every warning has a `kind` and `message` field.
+  # resolved warnings will be filtered
+  generateWarnings = pkg: let
+    issues = lib.filter (issue: !(issue.resolved or false)) (pkg.meta.issues or []);
+    maintainerless = if hasNoMaintainers pkg then [{
+      kind = "maintainerless";
+      message = "This package has no maintainers";
+    }] else [];
+  in
+    issues ++ maintainerless;
+  # Check whether this warning should be ignored
+  ignoreWarning = pkg: warning:
+    let
+      pkgName = lib.getName pkg;
+      # Check all pattern combinations that always apply
+      patternMatchesThisWarningBasic = pattern:
+        pattern == "*.*" ||
```

Comment:
```suggestion
        pattern == "*" ||
```

Is this covered?

Reply:
https://github.com/NixOS/nixpkgs/pull/177272#discussion_r981358796 @maralorn and I had a short discussion about this, and decided that it would be simpler to keep it out for now. Can always be added later on if needed though.

The current type description – which will show up in the error message when trying "*" – is `list of ("packageName.warningKind" or "packageName.*" or "*.warningKind" or "*.*")`, so it should be clear to users.